rmgpy/data/thermo.py

```diff
@@ -1404,7 +1405,7 @@ def computeGroupAdditivityThermo(self, molecule):
                     logging.error(molecule.toAdjacencyList())
                     raise
                 # Correct for gauche and 1,5- interactions
-                if not cyclic:
+                if molecule.isAtomInCycle(atom) and (atom.atomType.label!='Cb'):
```

Comment:
This work around seems too non-general to me. I think we should separate interactions between cyclic ones and non-cyclic ones. Maybe the gauche and ortho should be in separate files, and analogously for meta and int15